src/classes/Foo5.cls

```diff
@@ -0,0 +1,15 @@
+public class Foo5 {
+    void foo() {
+        
+        
+        
+        String myString = 'MY_HARDCODED_STRING';
+        Blob hardCodedIV = Blob.valueOf('Hardcoded IV 123');
+        Blob myKey = Blob.valueOf(myString);
+        Blob data = Blob.valueOf('Data to be encrypted');
+        Blob encrypted = Crypto.encrypt('AES128', myKey, hardCodedIV, data);
```

Comment:
This encryption key is not randomized. Please ensure all encryption keys are randomized, for example using methods such as `Crypto.generateAesKey`.